up42/__init__.py

```diff
@@ -21,83 +21,64 @@
 """
 
 
-from up42.asset import Asset
-from up42.auth import Auth
-from up42.catalog import Catalog
-from up42.initialization import (
-    initialize_asset,
-    initialize_catalog,
-    initialize_job,
-    initialize_jobcollection,
-    initialize_jobtask,
-    initialize_order,
-    initialize_project,
-    initialize_storage,
-    initialize_tasking,
-    initialize_webhook,
-    initialize_workflow,
+from up42 import asset
+from up42 import auth as up42_auth
```

Comment:
Why not just auth?

Reply:
although it's not necessary here, I wanted to be consisted like we imported it in [other modules.](https://github.com/up42/up42-py/blob/97f4092efebc4d6cf8713800eb24733f19d99163/up42/catalog.py#L14)